e2e/support/cypress_data.js

```diff
@@ -25,15 +25,15 @@ export const SAMPLE_DB_SCHEMA_ID = "1:PUBLIC";
 // Use only for e2e helpers and custom commands. Never in e2e tests directly!
```

Comment:
see https://github.com/metabase/metabase/pull/36677/files#r1427855029 for why I have this change